sources/install/package_base.sh

```diff
@@ -199,6 +199,22 @@ function install_gf() {
     add-to-list "gf,https://github.com/tomnomnom/gf,A wrapper around grep to avoid typing common patterns"
 }
 
+function install_cyberchef() {
+    colorecho "Installing CyberChef"
+    local last_version
+    last_version=$(wget https://github.com/gchq/CyberChef/releases/latest/download/CyberChef.zip 2>&1 | grep Location: | grep -E -o 'v.*' | cut -d '/' -f 1)
+
+    if [ ! -z "$last_version" ]; then
+        mkdir /opt/tools/CyberChef
+        wget https://github.com/gchq/CyberChef/releases/download/$last_version/CyberChef_$last_version.zip -O /tmp/CyberChef.zip
+        unzip -o /tmp/CyberChef.zip -d /opt/tools/CyberChef/
+        rm /tmp/CyberChef.zip
+        mv /opt/tools/CyberChef/CyberChef_$last_version.html /opt/tools/CyberChef/CyberChef.html
+        add-test-command "file /opt/tools/CyberChef/CyberChef.html"
+        add-to-list "CyberChef,https://github.com/gchq/CyberChef/,The Cyber Swiss Army Knife"
+    fi
```

Comment:
Once compiled, I opened the cyberchef.html file via x11 (firefox). I had the correct view of cyberchef and when I tried to convert to base64, x11 crashed and showed me the list of errors.

Reply:
@QU35T-code Sorry, but I'm not sure to understand what you tested, did you typed `firefox &> /dev/null &` and then opened the HTML file ? Because for me this seems to work.

Reply:
Run your script and execute: `firefox /opt/tools/CyberChef/CyberChef.html`
It launches correctly:

<img width="1638" alt="image" src="https://github.com/ThePorgs/Exegol-images/assets/51704860/bea62d7b-61a4-4b1e-9016-e8620b8d2630">

Then write something in the cyberchef input and try to convert to base64, for example. At that point, xquartz crashes. Could this be a Mac-only problem?

Reply:
Hey @QU35T-code,

I don't have the same problem as you (Exegol v4.2.5 and nightly image commit v.4ad9a812). Do you also have the same problem on the online version of CyberChef ?

Reply:
It'd be interesting to try through the desktop (beta) feature, to find out if it's related to X11

Reply:
With the desktop, it works, the problem is with X11. We can then resolve

<img width="1691" alt="image" src="https://github.com/ThePorgs/Exegol-images/assets/51704860/2fac4be7-9b24-4848-8fc8-7cc40555347b">